.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['2.5', '2.6', '2.7']
```

Comment:
Do we want to add 3.0 here as well?

Reply:
Definitely! I tried adding it, but there are some dependencies that aren’t compatible. Would it be cool to do that in a follow up PR?

Reply:
Ah, ok, thanks for the context!  I'm fine with that being a separate PR then.